### DISCUSSION TITLE
Add option to use MAMEUI

### DISCUSSION THREAD

**Discussion by paulmortlock** [sam-ludlow/mame-ao#21] [Ideas]
![image](https://github.com/sam-ludlow/mame-ao/assets/7123294/da4d5ec6-74f8-4522-8f59-0a21459f579d)

**Comment by sam-ludlow** [comment]
Hello @paulmortlock 

No plans, what does it do?

Cheers
Sam

**Reply by paulmortlock** [reply]
I think it is MAME but with a windows frontend not sure how it works but is mame.

**Comment by sam-ludlow** [comment]
I'm closing this, no plans to handle MAMUI, or any other front ends, MAME-AO is the front end.

I may do something with HBMAME one day, but that's it.
